frame/system/src/extensions/check_mortality.rs

```diff
@@ -26,17 +27,17 @@ use sp_runtime::{
 };
 
 /// Check for transaction mortality.
-#[derive(Encode, Decode, Clone, Eq, PartialEq)]
-pub struct CheckMortality<T: Config + Send + Sync>(Era, sp_std::marker::PhantomData<T>);
+#[derive(Encode, Decode, Clone, Eq, PartialEq, TypeInfo)]
+pub struct CheckMortality<T: Config + TypeInfo + Send + Sync>(Era, sp_std::marker::PhantomData<T>);
```

Comment:
I still wonder why we bound Send and Sync here, if extensions needs Config to be Send and Sync then we should bound Send and Sync on Config directly,
same goes for TypeInfo.
better to refactor those `TypeInfo + Send + Sync` in a follow up.

Reply:
Actually I can remove this `TypeInfo` bound now using `skip_type_params`

Reply:
yes it is even better if possible